src/components/v5/common/ActionSidebar/partials/forms/StagedPaymentForm/partials/StagedPaymentRecipientsField/StagedPaymentRecipientField.tsx

```diff
@@ -0,0 +1,120 @@
+import { CopySimple, Plus, Trash } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React, { type FC } from 'react';
+import { useFieldArray, useFormContext, useWatch } from 'react-hook-form';
+
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { useMobile, useTablet } from '~hooks';
+import { formatText } from '~utils/intl.ts';
+import useHasNoDecisionMethods from '~v5/common/ActionSidebar/hooks/permissions/useHasNoDecisionMethods.ts';
+import Table from '~v5/common/Table/Table.tsx';
+import Button from '~v5/shared/Button/Button.tsx';
+
+import { useStagedPaymentTableColumns } from './hooks.tsx';
+import {
+  type StagedPaymentRecipientsTableModel,
+  type StagedPaymentRecipientsFieldProps,
+  type StagedPaymentRecipientsFieldModel,
+} from './types.ts';
+
+const StagedPaymentRecipientsField: FC<StagedPaymentRecipientsFieldProps> = ({
+  name,
+}) => {
+  const hasNoDecisionMethods = useHasNoDecisionMethods();
+  const {
+    colony: { nativeToken },
+  } = useColonyContext();
+
+  const isTablet = useTablet();
+  const isMobile = useMobile();
+  const fieldArrayMethods = useFieldArray({
```

Comment:
If you want, you can actually strictly type this like for example:

```ts
type Person = {
  name: string;
}

type FormValues = {
  people: Person[]
}

const { control } = useFormContext<FormValues>();
const { fields } = useFieldArray({ control, name: 'this should auto complete' });
```

But I guess it depends on whether or not yup has correctly inferred your schema's type